app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -54,6 +54,7 @@ def paid_with_credit_card?
 
   def assign_attributes(with_save, attributes)
     subscription.assign_attributes(attributes)
+    subscription.user.cpf = subscription.donator_cpf
 
     subscription.save if with_save && !has_errors?
```

Comment:
it's definitely better right? But what do you think of also replace `!has_errors?` for `subscription.valid?` ?